src/Lib.hs

```diff
@@ -78,7 +79,9 @@ sma candles = SMA (totalSum candles / numCandles candles)
 
 -- | Using the old window data computes the next exponential moving average values
 getNextWindow :: ExchangeConf -> Window -> IO Window
-getNextWindow config oldWindow@(Window (shortEMA, longEMA) _) = do
+getNextWindow config oldWindow@(Window (shortEMA, longEMA) _) =
+  {- Making request to API will throw exception if over rate limit, just try again -}
+  flip catchIOError (\_ -> putStrLn "Failed request" >> return oldWindow) $ do
```

Comment:
I'm not too worried about this, I know that the error I'm catching here is a failed request because of the rate limit that I hit sometimes. The extra details in the error don't really help me. 

Yep 😈